03-programming.Rmd

```diff
@@ -798,6 +802,16 @@ A final option to use just-in-time (JIT) compilation. The `enableJIT()` function
 disables JIT compilation if the argument is `0`. Arguments `1`, `2`, or `3` implement
 different levels of optimisation. JIT can also be enabled by setting the environment
 variable `R_ENABLE_JIT`, to one of these values.
+
 ```{block, type="rmdtip"}
 We recommending setting the compile level to the maximum value of 3.
 ```
+
+The impact of compiling on install will vary from package to package: for packages
+that already have lots of pre-compiled speed gains will be small, as stated in the
+[R Installation and Administration manual](https://cran.r-project.org/doc/manuals/r-release/R-admin.html).
+
+```{block type="rmdwarning"}
+Not all packages work if compiled on installation.
```

Comment:
That's what it says in the manual: "Not all contributed packages work correctly when byte-compiled."

Source: https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Byte_002dcompilation